ui/src/components/DialogComponents/TitleSection.tsx

```diff
@@ -34,6 +43,24 @@ const TitleSectionComponent: React.FC<Props> = ({
           <XIcon strokeWidth={"3"} width={"28"} />
         </IconButton>
       </Stack>
+      {additionalInfo && (
+        <Alert severity="info" className={styles["info"]}>
+          {additionalInfo.message}{" "}
```

Comment:
Can we put this in typography and have the links on new lines?
```suggestion
          <Typography>{additionalInfo.message}</Typography>
```